lxd/instance/instance_utils.go

```diff
@@ -13,11 +13,12 @@ import (
 	"github.com/flosch/pongo2"
 	"github.com/pkg/errors"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```

Comment:
Looks like you're using an editor or static analysis tool which introduces the package name when it doesn't match with the import path. We usually don't like this and have been undoing such changes when they appear.